core/src/main/resources/error/error-classes.json

```diff
@@ -150,7 +150,12 @@
     "sqlState" : "0A000"
   },
   "UNSUPPORTED_FEATURE" : {
-    "message" : [ "The feature is not supported: %s" ],
+    "message": [
+      "The feature is not supported: %s"
+    ]
+  },
+  "UNSUPPORTED_INPUT_TYPE" : {
```

Comment:
Could you reuse the existing error class, please.
https://github.com/apache/spark/blob/b63a577b656b5fc56feef1666d1f4d1048b945fe/core/src/main/resources/error/error-classes.json#L148-L151

Reply:
@MaxGekk done, sorry for the delay, and would you please sherd some lights on how to write tests for the  `error class, sqlState and error message.` thanks.

Reply:
Could you look at the error classes PRs (https://issues.apache.org/jira/browse/SPARK-37935), for instance https://github.com/apache/spark/pull/35466